src/Info.ts

```diff
@@ -54,11 +54,11 @@ export default class Info {
         // if using .syntax we can only read notes
         let commandIsFetch = false;
 
-        // Needs to start with '/' or '!' or in separate cases '.'
+        // Needs to start with '/' / '!' or '.'
+        const contentPrefix = message.cleanContent[0];
```

Comment:
split[0][0]?

Reply:
This gets the first char in the string. So the use of that was unnecessary. But either could be used, just in my opinion, i thought it looked messy; Though, I am re-referencing the cleanContent, when the same data is in 'split'.